src/EditorFeatures/Core/FindUsages/FindUsagesContext.cs

```diff
@@ -27,6 +31,9 @@ protected FindUsagesContext()
 
         public virtual Task OnReferenceFoundAsync(SourceReferenceItem reference) => Task.CompletedTask;
 
-        public virtual Task ReportProgressAsync(int current, int maximum) => Task.CompletedTask;
+        protected virtual Task ReportProgressAsync(int current, int maximum) => Task.CompletedTask;
+
+        Task IFindUsagesContext.ReportProgressAsync(int current, int maximum)
```

Comment:
Should this method also be [Obsolete]ed?

Reply:
Obsoleting an explicit impl doesn't really every do anything... since you can't actually reference them :)